app/serializers/api/v2/changes_serializer.rb

```diff
@@ -0,0 +1,14 @@
+module Api
+  module V2
+    class ChangesSerializer
+      include JSONAPI::Serializer
+
+      set_type :change
```

Comment:
I think the type gets inferred from the class name if you don't specify it here? I think json api should serialize included types as the plural form, too - it just so happens that we're not completely compliant with our types for other serializers.